### ISSUE TITLE
Prey and Doom needs overlay enabled, or the game will stutter after 15-30 minutes

### ISSUE THREAD

**Issue by mikedebian** [ValveSoftware/Proton#751]
Dota 2 Vulkan also stutters after about 30 minutes. The apparent fix (according to dota2-vulkan github issue) is to enable steam overlay for said game. This also works in Doom and Prey (both users from NVidia and AMD have this issue).

I've had this issue myself with Dota 2, with different graphics card, CPU and distros, for over a year.

If using steamplay, a core gets stuck on 100% if not overlay is enabled, after about 15-30 minutes of gameplay. If overlay is enabled, the problem seems to go away.

Steam Beta
Arch Linux
Proton 3.74

Ryzen 1800x
Vega 56
Mesa GIT

**Comment by smirgol** [comment]
I've noticed something similar in Fallout 4, where is runs super smooth, then after about half an hour it suddenly starts to stutter. I'll check if this is also related to the disabled overlay.

**Comment by mikedebian** [comment]
@smirgol , please comment back if you have tested it.

**Comment by smirgol** [comment]
It's not clear to me yet if this is related to my stuttering in Fallout 4. Unfortunately I did not have that much time to play the game the last days. 
I did 2 sessions though, one with overlay enabled, one without, both at about 1.5h of length. With overlay disabled I got the stuttering, but I was alt-tabbing when it happened (and did so a couple of times before, without problems). So I'm not sure if the alt-tabbing triggered it or if it was just a coincidence and it's actually the overlay issue.
But none of my cores (8 in total) was stuck at 100% when it happened, so maybe that's an indication that it is a different problem.
